lib/matrix_reloaded/vector.ex

```diff
@@ -65,7 +65,7 @@ defmodule MatrixReloaded.Vector do
       [1, 2, 3]
 
   """
-  @spec transpose(t()) :: t()
```

Comment:
What is wrong here?

Reply:
A better solution could be the definition of a vector as a column vector and vector, eg. `@type column() :: [t()]` (or better name `@type transposed() :: [t()]` ???) and `@type t() :: [number]`. Or derive type of the vector from the matrix. The change of type is quite unclear and complicates the readability.

Reply:
Oki, that sounds better. I'll fix it.